docs/tcti.rst

```diff
@@ -10,5 +10,9 @@ TCTI
    :undoc-members:
 
 .. automodule:: tpm2_pytss.PyTCTI
-   :members: TCTILdr
+   :members: PyTCTI
+   :undoc-members:
```

Comment:
I thought commit "docs: fix PyTCTI members" would have fixed the documentation not showing up correctly for PyTCTI but it doesn't seem too, hrmm.

Reply:
Ahh that makes sense, auto-module is on the package not the classes. Thanks.